src/forms.py

```diff
@@ -27,12 +30,13 @@ class OrderSubscriptionForm(FlaskForm):
 
 
 class OrderPacketsForm(FlaskForm):
-    email = StringField(label='Email', validators=[DataRequired(), Email()])
+    email = SelectField(label='Email', choices=([(None, "Current user")]))
```

Comment:
Currently, if the user is an admin all filled-in email fields (e.g. email field in edit profile) will be populated with the existing users' emails, but if the user is a normal user the email field will be populated with 'Current User' instead of their email. Is that intended behaviour?

Reply:
Yes that is intended behaviour, but feel free to suggest any changes to this design

Reply:
After further review, I've come to the conclusion that it wasn't intended. The form returning 'Current User' isn't accounted for, leading to an unhandled exception.

Reply:
But although design is intended if you found an execution path that leads to unhandled exception then please provide reproduction steps